cupy/cuda/runtime.pyx

```diff
@@ -234,6 +236,21 @@ cpdef int deviceGetAttribute(int attrib, int device) except? -1:
     check_status(status)
     return ret
 
+cpdef int deviceGetByPCIBusId(str pci_bus_id) except? -1:
+    # Encode the python string before passing to native code
+    byte_pci_bus_id = pci_bus_id.encode("UTF-8")
```

Comment:
How about `ascii` instead of `utf-8`?

Reply:
Yes, I guess the string will only ever need to be ASCII